jdaviz/configs/default/plugins/export/export.py

```diff
@@ -389,6 +406,38 @@ def save_movie(self, viewer, filename, filetype, i_start=None, i_end=None, fps=N
 
         return filename
 
+    def save_subset_as_region(self, selected_subset_label, filename):
+        """
+        Save a subset to file as a `Region` object in the working directory.
+        Currently only enabled for non-composite spatial subsets. Can be saved
+        as a `.fits` or `.reg` file. If link type is currently set to 'pixel',
+        then a pixel region will be saved. If link type is 'wcs', then a sky
+        region will be saved. If a file with the same name already exists in the
+        working directory, it will be overwriten.
+        """
+
+        # type of region saved depends on link type
+        link_type = getattr(self.app, '_link_type', None)
+
+        region = self.app.get_subsets(subset_name=selected_subset_label,
+                                      include_sky_region=link_type == 'wcs')
+
+        if len(region) > 1:
+            raise NotImplementedError("Export not yet supported for composite subsets.")
+
+        region = region[0][f'{"sky_" if link_type == "wcs" else ""}region']
+
+        try:
+            region.write(filename, overwrite=True)
+        except Exception as e:
+            self.hub.broadcast(SnackbarMessage(
+                f"{selected_subset_label} failed to export to {str(filename)}: {e}",
+                sender=self, color="error"))
+        finally:
+            self.hub.broadcast(SnackbarMessage(
+                f"{selected_subset_label} exported to {str(filename)}",
+                sender=self, color="success"))

```

Comment:
once this is rebased onto the latest in #2755, the snackbars will be handled by the `vue_export_from_ui` method and should be removed from here.

Reply:
done